Stratis.Bitcoin/Features/Wallet/Wallet.cs

```diff
@@ -107,6 +107,23 @@ public IEnumerable<Script> GetAllPubKeysByCoinType(CoinType coinType)
                 yield return address.ScriptPubKey;
             }
         }
+
+        /// <summary>
+        /// Adds an account to the current wallet.
+        /// </summary>
+        /// <remarks>The name given to the account is of the form "account (i)" by default, where (i) is an incremental index starting at 0.
+        /// According to BIP44, an account at index (i) can only be created when the account at index (i - 1) contains at least one transaction.
```

Comment:
Maybe add to the comment that this method assumes that the BIP44 requirement of account i-1 has transactions